lncfg/address.go

```diff
@@ -121,6 +121,12 @@ func IsLoopback(addr string) bool {
 	return false
 }
 
+// isIPv6Host returns true if the host is IPV6 and false otherwise.
+func isIPv6Host(host string) bool {
+	// An IPv4 host without the port shouldn't have a colon.
+	return strings.Contains(host, ":")
```

Comment:
Wouldn't it need to also contain a `[` and `]` at this point to be valid? At least the `net.SplitHostPort()` function expects it that way. Perhaps we should add that to the check?

Reply:
I believe a more correct method would be to `net.ParseIP` to parse and if not `nil` then check for `:`

Reply:
I believe `net.SplitHostPort` removes the brackets and then it's just the raw host. IPv4 would not have a colon in the raw host.